python-test/features/steps/control_plane_agents.py

```diff
@@ -152,6 +153,15 @@ def check_agent_tags(context, amount_of_tags):
     assert_that(len(dict(agent["orb_tags"])), equal_to(int(amount_of_tags)), "Amount of orb tags failed")
 
 
+@step("remotely restart the agent")
+def reset_agent_remotely(context):
```

Comment:
it would be good to add the Then/When/And even if in a comment above to identify faster when reading